services/githubService.js

```diff
@@ -121,19 +108,28 @@ const fetchStalePRs = async () => {
 
 /**
  * Fetches the latest 10 open PRs
- * @todo fetch N from query params
+ * @param pageNumber pagination page number
+ * @param perPage number of entries per page
+ * @param isOpen boolean checking for open or stale PRs
  */
-const fetchOpenPRs = async (pageNumber) => {
+const fetchPRs = async ({ pageNumber = 1, perPage = 10, isOpen = true, username = null }) => {
   try {
-    const url = getGithubURL({
-      is: 'open'
-    }, {
-      sort: 'created',
-      order: 'desc',
-      per_page: 10,
-      page: pageNumber
-    })
-    return getFetch(url)
+    const searchParams = { is: 'open' }
+    if (username) {
+      const user = await fetchUser({ username })
+      searchParams.author = user.github_id
+    }
+
+    const url = getGithubURL(
+      searchParams,
+      {
+        sort: 'created',
+        order: isOpen ? 'desc' : 'asc',
```

Comment:
Now I see why was `isOpen: false` somewhere in other file
Wrong choice of variable name causing confusions 😛